components/molecules/TopUsersPanel/top-user-panel.tsx

```diff
@@ -1,15 +1,29 @@
 import React from "react";
-import TopUserCard, { TopUserCardProps } from "components/atoms/TopUserCard/top-user-card";
+import TopUserCard from "components/atoms/TopUserCard/top-user-card";
+import { useFetchTopUsers } from "lib/hooks/useFetchTopUsers";
+import SkeletonWrapper from "components/atoms/SkeletonLoader/skeleton-wrapper";
 
 interface TopUsersPanelProps {
-  users: TopUserCardProps[];
+  loggedInUserLogin: string;
 }
-const TopUsersPanel = ({ users }: TopUsersPanelProps) => {
+const TopUsersPanel = ({ loggedInUserLogin }: TopUsersPanelProps) => {
+  const { data, isLoading } = useFetchTopUsers();
+
+  const topUsersWithoutLoggedInUser = data ? data.filter((user) => user.login !== loggedInUserLogin) : [];
+  const top3Users = topUsersWithoutLoggedInUser.slice(0, 3).map((user) => user.login);
+
   return (
     <div className="flex flex-col max-w-xs gap-6 p-6 bg-white border rounded-xl">
       <h2 className="pb-2 text-2xl border-b">Top Users</h2>
```

Comment:
```suggestion
      <h2 className="pb-2 text-2xl border-b">Top Contributors</h2>
```

I know we got this far with it showing "Top Users," but I would like to propose use rename this to "Top Contributors"

## Reasoning for contributor:

Users is not a great or descriptive term. Contributors is better, because it describes our audience and the what someone who signs up can strive to become. 

We also have the chance to change how the industry looks at open source and the definition of a contributor. Code contributions is not he end goal and individuals who write blog posts about a repo will be labeled as an "open source contributor."

I also think Contributor is a way to take a bigger view on how many people can impact a project in many way. For example p5.js includes Designers, Project Managers, and other coding adjacent roles in their allcontributors section. 

https://github.com/processing/p5.js#contributors

Would love to know your thoughts @BekahHW

Reply:
I like this suggestion.

Like you said, it's in line with the target audience of people on the Highlights page: the venn-diagram of people who are adding value to open source (in code, with content, reviewing changes, by answering questions, triaging, etc.) and what a "contributor" is ends up just being a circle.

Reply:
Exactly!

When we talk Open Source, Contributors should be what comes to mind first before anything else.

That's Inclusive!